examples/autoscaler-webhook/Makefile

```diff
@@ -31,9 +31,9 @@ project_path := $(dir $(mkfile_path))
 root_path := $(realpath $(project_path)/../..)
 
 ifeq ($(REPOSITORY),)
-	autoscaler_webhook_tag := autoscaler-webhook:0.10
+	autoscaler_webhook_tag := autoscaler-webhook:0.11
```

Comment:
Note: As a separate PR, will need to update the e2e tests with the new version once published.

https://github.com/googleforgames/agones/blob/eb4035e1f78e7a8e0c147a013a5405f904ab9899/test/e2e/fleetautoscaler_test.go#L647

Same with simple-game-server.

Although there seem to be so many changes in PRs recently, might be better to make that change to the e2e tests once they all finish landing.

Reply:
If this looks good, I will push the images to prod and then update other files(e2e tests) in this same PR, tomorrow. WDYT?

Reply:
Friendly reminder, does this look good to you, @igooch?

Reply:
@igooch, Please feel free to merge this PR if the changes look good to you.😀